integrations/airflow/tests/mocks/git_mock.py

```diff
@@ -11,15 +11,19 @@
 # limitations under the License.
 
 import logging
-
-from marquez_airflow.utils import execute_git
```

Comment:
Having this resulted in cyclical dependencies.

Reply:
Good catch 👍